funtofem/model/function.py

```diff
@@ -206,6 +206,25 @@ def get_gradient_component(self, var):
 
         return 0.0
 
+    def set_timing(self, start: int = None, stop: int = None, averaging: bool = None):
+        """
+        Set the time window of function evaluation for FUN3D unsteady
+        analysis, only do this for aero functions
+
+        Parameter
+        ---------
+        start: int
+            starting time index
+        stop: int
+            final time index (note you need to specify this for unsteady)
+        averaging: bool
+            whether or not the aero function is evaluated
+        """
```

Comment:
```suggestion
        """
        if self.analyis_type != "aerodynamic"
            return self
```

Reply:
If this should only be used for aero functions, why not add a check to make sure?